tests/acceptance/nested-addons-smoke-test-slow.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+var path       = require('path');
+var fs         = require('fs-extra');
+
+var runCommand          = require('../helpers/run-command');
+var acceptance          = require('../helpers/acceptance');
+var copyFixtureFiles    = require('../helpers/copy-fixture-files');
+var createTestTargets   = acceptance.createTestTargets;
+var teardownTestTargets = acceptance.teardownTestTargets;
+var linkDependencies    = acceptance.linkDependencies;
+var cleanupRun          = acceptance.cleanupRun;
+
+var chai = require('../chai');
+var expect = chai.expect;
+var file = chai.file;
+var dir = chai.dir;
+
+var appName  = 'some-cool-app';
+
+describe('Acceptance: nested-addons-smoke-test', function() {
+  this.timeout(360000);
+
+  before(function() {
+    return createTestTargets(appName);
```

Comment:
since this part here takes quite a long time it might be beneficial to combine this with other "slow" tests. e.g. if there is something like an `addons-smoke-test-slow` already then you should probably just add it there instead.